pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -36,28 +36,29 @@ var serverName string
 var serverNameAndPort string
 
 type getconfigContext struct {
-	zedagentCtx              *zedagentContext // Cross link
-	ledManagerCount          int              // Current count
-	configReceived           bool
-	configGetStatus          types.ConfigGetStatus
-	updateInprogress         bool
-	readSavedConfig          bool
-	configTickerHandle       interface{}
-	metricsTickerHandle      interface{}
-	pubDevicePortConfig      *pubsub.Publication
-	pubPhysicalIOAdapters    *pubsub.Publication
-	devicePortConfig         types.DevicePortConfig
-	pubNetworkXObjectConfig  *pubsub.Publication
-	subAppInstanceStatus     *pubsub.Subscription
-	subNodeAgentStatus       *pubsub.Subscription
-	pubZedAgentStatus        *pubsub.Publication
-	pubAppInstanceConfig     *pubsub.Publication
-	pubAppNetworkConfig      *pubsub.Publication
-	pubCertObjConfig         *pubsub.Publication
-	pubBaseOsConfig          *pubsub.Publication
-	pubDatastoreConfig       *pubsub.Publication
-	pubNetworkInstanceConfig *pubsub.Publication
-	rebootFlag               bool
+	zedagentCtx              	*zedagentContext // Cross link
+	ledManagerCount          	int              // Current count
+	configReceived           	bool
+	configGetStatus          	types.ConfigGetStatus
+	updateInprogress         	bool
+	readSavedConfig          	bool
+	configTickerHandle			interface{}
+	metricsTickerHandle			interface{}
+	pubDevicePortConfig			*pubsub.Publication
+	pubPhysicalIOAdapters		*pubsub.Publication
+	devicePortConfig			types.DevicePortConfig
+	pubNetworkXObjectConfig		*pubsub.Publication
+	subAppInstanceStatus		*pubsub.Subscription
+	subNodeAgentStatus			*pubsub.Subscription
+	pubZedAgentStatus 			*pubsub.Publication
+	pubAppInstanceConfig		*pubsub.Publication
+	pubAppNetworkConfig      	*pubsub.Publication
+	pubCertObjConfig         	*pubsub.Publication
+	pubBaseOsConfig          	*pubsub.Publication
+	pubDatastoreConfig       	*pubsub.Publication
+	pubNetworkInstanceConfig 	*pubsub.Publication
+	rebootFlag               	bool
+	globalItemMap				*types.ConfigItemSpecMap
```

Comment:
"configItemSpec" is a better name than "globalitemMap" ? Because it has both GlobalConfigIitems as well as per-agent config items spec??